core/schema-management.md

```diff
@@ -0,0 +1,40 @@
+# Schema Management
+
+## Problem Description
+
+To enable data exchange between business parties involved in a Baseline Workflow (or only Workstep?) (in short, parties), there must be present a set of pre-agreed schemas. 
+
+Furthermore, at least identifying different versions of the same schema must be possible for each Baseline instance. A Baseline instance may support different versions of the same schema.
+
+## Schema
+  
+Minimum data model describing the structure of exchanged messages between different components and APIs. It must include list of mandatory fields/properties and their types (and their description?).
+
+## Agreement on Schema
+(in scope?)
+An agreement could be _explicit_ with stating the list of agreed upon versioned schemas (e.g. structured as a Merkle Tree with schemas as leaves) by all the parties or _implicit_ by their implementation. 
+
+(question: An explicit agreement could be seen as a commitment which makes Baseline processes (at least the intial setup) more expensive where an implicit one could be seen as a set of supported APIs which is already live but poses runtime incompatibility.)
+
+## Reaching an Agreement on Schema
+(in scope?)
+
+## Managing Different Schemas
+
+All the schemas must be available to all the Baseline instances involved in a Workflow (or only Workstep?).
+
+Schemas may be available publicly to everyone.
+
+A Schema must be uniquely identifiable by the Baseline instances.
```

Comment:
I'm unclear on the context of this requirement.

Reply:
My understanding is that a Baseline Protocol Instance will be reused for different workgroups/steps. Each group/step will use different schema(s). That's why we need a component to manage these schemas and also they must be uniquely identifiable to be addresses. Isn't this the main reason to have this brainstorming in the first place?

Reply:
See above response @humbitious